packages/suite/src/actions/suite/analyticsActions.ts

```diff
@@ -262,6 +262,12 @@ export type AnalyticsEvent =
               type: 'bug' | 'suggestion';
           };
       }
+    | {
+          type: 'guide/tooltip';
```

Comment:
Sorry, I know that it is my wording 😀, but I would modify it to something more obvious. This sounds like some tooltip click inside the guide panel. If you have better wording, feel free to use it.

```suggestion
          type: 'guide/tooltip-link/navigation';
```

Please bump analytics version and add it to changelog. (I will update analytics docs with description what to do when you need to add new event)